web/modules/custom/do_ai_alt_text/tests/src/Kernel/AltTextGeneratorTest.php

```
@@ -0,0 +1,256 @@
+    $this->chatFailure = new \RuntimeException('Quota exceeded.');
+    $this->chatFailsAtCall = 2;
+
+    // Assert.
+    $this->expectException(AltTextGenerationException::class);
+
+    // Act.
+    try {
+      $this->generator()->regenerateForMedia($media);
+    }
+    finally {
+      $reloaded = Media::load($media->id());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\media\Entity\Media' in method 'testRegenerateForMediaLeavesEntityUntouchedOnFailure'. (undefined)

(StaticAccess)